test/verify.tests.js

```diff
@@ -69,6 +69,30 @@ describe('verify', function() {
     });
   });
 
+  describe('option: complete', function() {
```

Comment:
Sorry, I didn't realize about this before, can you follow similar approach as this option? 
https://github.com/auth0/node-jsonwebtoken/blob/master/test/option-nonce.test.js

That is: `option-complete.test.js`, using `testUtils.asyncCheck` (to report errors correctly) and  `testUtils.verifyJWTHelper` (to verify sync and async).

We are refactoring the tests moving towards that model so each option and claim has its own tests using common tooling.

Reply:
Moved here https://github.com/auth0/node-jsonwebtoken/pull/522/commits/4d8f50af92b12b287f04f00e0c71dd2929ce5ba4

Apart from that, I added another unit test for `complete: false` option.